x/auth/stdtx.go

```diff
@@ -150,6 +150,11 @@ func (fee StdFee) Bytes() []byte {
 	return bz
 }
 
+// Bytes for signing later
```

Comment:
```suggestion
// GasPrices returns the gas prices for a StdFee.
//
// NOTE: The gas prices returned are not the true gas prices that were
// originally part of the submitted transaction because the fee is computed
// as fee = ceil(gasWanted * gasPrices).
```